components/project/journey.js

```diff
@@ -0,0 +1,50 @@
+import styled from 'styled-components'
+
+import {media, sizes} from '../../styles/media'
+import Post from '../post'
+import AddPost from '../add-post'
+import Card from '../card'
+
+const DesktopSlide = styled.div`
```

Comment:
Should this be called `PhoneSlide`? It seems to be hidden on desktop and visible on mobile